solidus_extension_dev_tools.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'ffaker', '~> 2.13'
   spec.add_dependency 'github_changelog_generator', '~> 1.15'
   spec.add_dependency 'rspec-rails', '~> 4.0.0.beta3'
+  spec.add_dependency 'rspec_junit_formatter'
```

Comment:
Can you lock the version here?

Reply:
I think it's better to leave it as it is and let rubygems match it with a compatible version of Rspec